exercises/affine-cipher/description.md

```diff
@@ -0,0 +1,54 @@
+Create an implementation of the affine cipher,
+an ancient encryption system created in the Middle East.
+ 
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with
+a mathematical function and then converted to the letter relating to
+its new numeric value. Although all monoalphabetic ciphers are weak,
+the affine cypher is much stronger than the atbash cipher,
+because it has many more keys.
+ 
+the encryption function is:
+ 
+  **E(x) = (ax + b) % m**
+  -  where `x` is the letter's index from 0 - length of alphabet - 1
+  -  `m` is the length of the alphabet - roman alphabet = 26
```

Comment:
Presumably `m` is not the difference between the length of the alphabet and the roman alphabet. Consider rephrasing:

> `m` is the length of the alphabet. For the roman alphabet, `m == 26`.

Reply:
I agree this would be a good change.  That line kept looking funny to me.